encord/http/utils.py

```diff
@@ -158,40 +151,27 @@ def _get_signed_url(
 
 
 def _upload_single_file(
-    file_path: str,
-    signed_url: dict,
-    pbar,
-    content_type: str,
-    max_retries: int,
-    backoff_factor: float,
+    file_path: str, signed_url: dict, pbar, content_type: str, *, max_retries: int, backoff_factor: float
```

Comment:
classic move by people with wide screens :D

Reply:
Have you seen my screens? :D